doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -154,6 +154,18 @@ nested into scopes, by relating operators to their parent operators.
 | `parent_id` | [`bigint`] | The ID of the operator's parent operator.                                                   |
 | `worker_id` | [`bigint`] | The ID of the worker thread hosting the operators.                                          |
 
+### `mz_dataflow_arrangement_sizes`
+
+The `mz_dataflow_arrangement_sizes` view describes how many records and batches
+are contained in operators under each dataflow.
+
+| Field     | Type       | Meaning                                                                      |
+|-----------|------------|------------------------------------------------------------------------------|
+| `id`      | [`bigint`] | The ID of the [dataflow]. Corresponds to [`mz_dataflows.id`](#mz_dataflows). |
+| `name`    | [`bigint`] | The name of the object (e.g. index) maintained by the dataflow.              |
+| `records` | [`bigint`] | The number of records in all arrangements in the dataflow.                   |
+| `batches` | [`bigint`] | The number of batches in all arrangements in the dataflow.                   |
```

Comment:
I can't find any reference to "batches" in the context of "arrangements" in our docs. Is this the same as "collections"?

Reply:
No, it's not. I am not sure if we publicly document this concept anywhere, but I kept it in here because it's already present in `mz_arrangement_sizes`. So I think our lack of documentation for what "batches" means is a separate issue.